frontend/packages/dev-console/src/utils/create-knative-utils.ts

```diff
@@ -1,5 +1,5 @@
 import { k8sCreate, K8sResourceKind } from '@console/internal/module/k8s';
-import { KsServiceModel } from '../models';
+import { ServiceModel } from '@console/knative-plugin';
```

Comment:
Shall we move this util as well under knative-plugin, WDYT?

Reply:
We could. Since the util isn't being used yet and after our discussion I wasn't sure if it was going to be updated to align more closely with the forms apis or not.
For now I'll leave it here and you can move it when you get to completing the work on the import forms.